server/src/main/java/com/paypal/selion/grid/SelendroidJarSpawner.java

```diff
@@ -44,7 +44,7 @@ public SelendroidJarSpawner(String[] args, ProcessLauncherOptions options) {
         setType(InstanceType.SELENDROID);
     }
 
-    public static final void main(String[] args) {
+    public static void main(String[] args) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1537384866/issues/source?bid=2928425&fileBranchId=2989551#l47)